File: python/01_at_least_one_true.py
"""
¿Al menos uno es verdadero?
Fácil
10 pts
Booleanos

Enunciado
Dados dos booleanos, devuelve True si al menos uno de los dos es True (compuerta OR).

Ejemplo
at_least_one_true(True, False)  → True
at_least_one_true(False, False) → False
at_least_one_true(True, True)   → True
Puedes usar print() para depurar. Los resultados aparecen en la Consola de salida, no en el navegador.
"""


def at_least_one_true(first_value: bool, second_value: bool) -> bool:
    # inclusive OR
    return first_value or second_value


print(at_least_one_true(True, False))  # True
print(at_least_one_true(False, False))  # False
print(at_least_one_true(True, True))  # True

File: typescript/01_at_least_one_true.ts
// ¿Al menos uno es verdadero?
// Fácil
// 10 pts
// Booleanos

// Enunciado
// Dados dos booleanos, devuelve true si al menos uno de los dos es true (compuerta OR).

// Ejemplo
// atLeastOneTrue(true, false)  → true
// atLeastOneTrue(false, false) → false
// atLeastOneTrue(true, true)   → true
// Puedes usar console.log() para depurar. Los resultados aparecen en la Consola de salida, no en el navegador.

function atLeastOneTrue(firstValue: boolean, secondValue: boolean): boolean {
  return firstValue || secondValue;
}

// No modificar: necesario para evaluar el resultado.
export { atLeastOneTrue };
